cypress.config.js

```diff
@@ -0,0 +1,7 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'https://bekkholt.github.io/social-media-client/',
```

Comment:
```suggestion
```

You should never test a remote URL, as the purpose of a test is to make sure the code in the branch is working. In this case you only test what is on the main branch deployed to the internet. This means that your tests provide 0% coverage no matter how many you write or how good they are.